swc/private/swc.bzl

```diff
@@ -66,6 +66,10 @@ https://docs.aspect.build/rulesets/aspect_rules_js/docs/js_library#data for more
     "root_dir": attr.string(
         doc = "a subdirectory under the input package which should be consider the root directory of all the input files",
     ),
+    "experimental_emit_isolated_dts": attr.bool(
+        doc = "Emit .d.ts files for TypeScript sources",
```

Comment:
add "instead of .js outputs" ?

Reply:
Also please link to SWC docs or discussion threads about the feature to save the reader some effort in researching this new capability

Reply:
There really is no documentation at the moment other then the commit in the changelog: https://github.com/swc-project/swc/blob/main/CHANGELOG.md#164---2024-06-22